### ISSUE TITLE
disassemble -d -c "bytecode": swarm hash in bytecode misinterpreted as INVALID opcode

### ISSUE THREAD

**Issue by AlexXiong97** [b-mueller/mythril#19]
**Exactly the same issue** with [another issue I reported for `manticore` project](https://github.com/trailofbits/manticore/issues/527), but the underlying reason/problem is the same, hope it helps.

Specifically, to reproduce such bug, (just an example):
1. Copy the [code of any contract code](https://etherscan.io/address/0x2a0c0dbecc7e4d658f48e01e3fa353f44050c208#code), save as local file.
2. run `solc --bin fileName.sol`, copy the bytecode and feed into `myth -d -c _bytecode_` .
3. Result: noticing the output ends with 
```shell
14289 STOP
14290 LOG1
14291 PUSH6 0x627a7a723058
14298 SHA3
14299 MULMOD
14300 INVALID
14301 MSTORE
14302 GASPRICE
14303 INVALID
14304 PUSH14 0xa9dbe23d1653684c888c4d7719d9
14319 INVALID
14320 INVALID
14321 INVALID
14322 NUMBER
14323 SSTORE
14324 INVALID
14325 AND
14326 INVALID
14327 MULMOD
14328 SLOAD
14329 EQ
14330 SWAP12
14331 STOP
14332 INVALID
```
After STOP, everything after that should be swarm hash, instead of "invalid" 

Thanks!

**Comment by b-mueller** [comment]
Hi @AlexXiong97, thanks for the feedback, I've been aware of the issue but ignored it so far :) Will try to fix it with one of the next releases!

**Comment by b-mueller** [comment]
Finally! Fixed in 0.8.22.
